relay/components/Projects/EditProjectMetaForm.tsx

```diff
@@ -61,6 +62,18 @@ const EditProjectMetaForm: React.FC<EditProjectMetaFormProps> = (props: EditProj
                                           transition ease-in-out duration-150" />
                           </div>
                       </div>
+
+                      {!props.hasReleased && <div className="sm:col-span-2">
+                        <label className="block">
+                          <span className="text-gray-700">Project Type</span>
+                          <select value={projectType} onChange={(e) => setProjectType(e.target.value)}
+                            className="form-select block w-full mt-1">
+                            <option value="node">node</option>
```

Comment:
This should probably be consistent with what we have in the Create Project page: https://github.com/valist-io/valist/blob/dfecfb2a3d364b6113dc652b9289aec9f2658757/relay/components/Projects/CreateProjectForm.tsx#L87

Reply:
Good catch!